### ISSUE TITLE
Addison's Disease no longer shows up in the autocomplete

### ISSUE THREAD

**Issue by TranslatorIssueCreator** [NCATSTranslator/DataGaps#8]
## Steps to reproduce:
I can't find "Addison's Disease" in the autocomplete on http://transltr-bma-ui-dev.ncats.io/ any longer.

**Comment by sierra-moxon** [comment]
<img width="1214" alt="Screen Shot 2022-09-26 at 3 57 31 PM" src="https://user-images.githubusercontent.com/4561292/192395116-f41a32d2-54af-4aa3-93b0-44e5e0d7abd5.png">
<img width="1233" alt="Screen Shot 2022-09-26 at 3 57 14 PM" src="https://user-images.githubusercontent.com/4561292/192395120-095bbc98-e64e-
<img width="1258" alt="Screen Shot 2022-09-26 at 3 56 56 PM" src="https://user-images.githubusercontent.com/4561292/192395133-4ddb82e0-00b8-43b6-b7b3-ed808197a160.png">
4923-a133-d40e0a7d9a87.png">

**Comment by sierra-moxon** [comment]
<img width="1258" alt="Screen Shot 2022-09-26 at 3 56 56 PM" src="https://user-images.githubusercontent.com/4561292/192395177-bad17ab1-6eee-4ec9-ac6d-00d0da126048.png">

**Comment by dnsmith124** [comment]
The original problem has been resolved--the dev instance of Node Normalizer was being used by the UI instead of the prod instance. I've pushed a change to correct that to the UI's frontend repo that will go out with the next build.  

As an aside, there's a little bit of a mix up regarding Addison's disease. The main curie for that (MONDO:0009410) is now obsolete in mondo. There may be a mix up with where the name resolver may not be updated to be in sync with the new node normalizer, so that may cause an issue with that particular curie. 

Either way, after the next build of the UI searching for 'addisons' or 'addison's' should return the following: 

- "pernicious anemia" - MONDO:0008228 
- "Addison disease" - MONDO:0009410
